### ISSUE TITLE
원하는 책 구매

### ISSUE THREAD

**Issue by BonnieSon** [BonnieSon/markbook#13]
### 요약

고객은 상세 페이지에서 원하는 책 구매 버튼을 누르면 해당 책 구매 사이트로 이동한다.

### 원하는 책 구매 시나리오

GIVEN: 로그인한 사용자는,
GIVEN: 상세 페이지에서,
WHEN: 책 구매 버튼을 누르면
THEN: 원하는 책 구매 사이트로 이동한다.

**Comment by yourhrh** [comment]
제 생각엔 책구매 링크 하나로 묶는거 보다는
원하는 구매 사이트를 누르면 거기로 이동하는 방식은 어떨까요?!

**Comment by BonnieSon** [comment]
네 맞아요. 제가 그렇게 구성하고 싶었는데 정리를 하다보니 하나로 묶였네요ㅎㅎ 동휘님이 제안해주신걸로 수정하겠습니다.
